model/installation_request.go

```diff
@@ -460,9 +462,12 @@ func (p *PatchInstallationRequest) addMissingIngressSourceRanges(installation *I
 	}
 
 	allowedIPRanges := strings.Split(installation.AllowedIPRanges, ",")
+
 	for _, ip := range ips {
-		if !contains(allowedIPRanges, ip) {
+		if IsIPRangeValid(ip) && !contains(allowedIPRanges, ip) {
 			allowedIPRanges = append(allowedIPRanges, ip)
+		} else if !IsIPRangeValid(ip) {
+			logrus.Errorf("Invalid IP range will not be appended: %s", ip)
```

Comment:
I think we still need a return here. Something that will cause a 400 to be returned to the caller